swan-lake/learn/quick-tour.md

```diff
@@ -56,122 +106,62 @@ You get the following output.
 [ballerina/http] started HTTP/WS listener 0.0.0.0:9090
 ```
 
-This means your service is up and running. You can invoke the service using an HTTP client. In this case, we use cURL.
+This means your service is up and running. 
 
-```bash
-$ curl http://localhost:9090/hello/sayHello
-```
+> **Note:** You can test the service by invoking it using an already-available HTTP client. For example, execute the command below in a new CLI tab to use [cURL](https://curl.haxx.se/download.html) as the client.
 
-> **Tip**: If you do not have cURL installed, you can download it from [https://curl.haxx.se/download.html](https://curl.haxx.se/download.html).
-
-You get the following response.
-
-```
-Hello Ballerina!
+```bash
+curl http://localhost:9090/hello/sayHello
 ```
 
-Alternatively, you can use a Ballerina HTTP client to invoke the service.
+Alternatively, you can create a Ballerina HTTP client and use that to invoke the service as follows.
 
-## Using a Client to Interact with a Network-Accessible Service
-
-A Ballerina client is a component, which interacts with a network-accessible service. It aggregates one or more actions that can be executed on the network-accessible service and accepts configuration parameters related to the network-accessible service.
+### Creating a 'helloClient' to Invoke the 'hello' Service
 
 There are two kinds of clients in Ballerina, inbound (or ingress) and outbound (or egress) clients. An outbound client object can be used to send messages to a network service.
 
-Having said that, let's see how you can use a Ballerina client to invoke the Hello World service.
-
-First, you need to create the client with the relevant endpoint URL as follows. We will use a Ballerina program with a `main` function, which will perform the invocation.
-
-> **Note**: returning `error?` allows you to use the `check` keyword to avoid handling errors explicitly. This is only done to keep the code simple. However, in real production code, you may have to handle those errors explicitly.
-
-```ballerina
-http:Client helloClient = new("http://localhost:9090/hello");
-```
-
-As the next step, add the below code to do a `GET` request to the Hello World service.
-
-```ballerina
-http:Response helloResp = check helloClient->get("/sayHello");
-```
-
-The remote call would return an `http:Response` if successful, or an `error` on failure. If successful, attempt retrieving the payload as a `string` and print the payload.
+Create an outbound Ballerina client as a Ballerina program with a `main` function with the content below to invoke the `hello` service.   
 
-```ballerina
-io:println(check helloResp.getTextPayload());
-```
-
-The complete source code should look similar to the following:
+> **Note**: Returning `error?` allows you to use the `check` keyword to avoid handling errors explicitly. This is only done to keep the code simple. However, in real production code, you may have to handle those errors explicitly.
 
+***hello_client.bal***
```

Comment:
Do we really want another client creation example in quick tour? Having one example (`Sunrise API Client`) is enough IMO.

Reply:
Removed the Sunrise API related use case. Now, we have a simple and easy-to-follow structure and flow explaining the use case in a straightforward manner.